runners/spark/src/main/java/org/apache/beam/runners/spark/translation/SparkContextFactory.java

```diff
@@ -93,6 +94,8 @@ private static JavaSparkContext createSparkContext(SparkContextOptions contextOp
       }
 
       conf.setAppName(contextOptions.getAppName());
+      conf.set("spark.serializer", KryoSerializer.class.getName());
+      conf.set("spark.kryo.registrationRequired", "true");
```

Comment:
can the kryo reference tracking feature also be disabled?

Reply:
not valid anymore. Only added classes for kryo registration, default is still JavaSerializer.